### PR TITLE
Update documented performance results

### PR DESCRIPTION
**Pull request by Sindica**

### CODE REVIEW

File: resource-management/docs/performance/perf-results.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+|    us west2-a |               LA, CA |
+|    us west3-c | Salt Lake city, Utah |
+|    us west4-a |    las Vegas, Nevada |
+
+* Schedulers:
+
+|     Region |             Location |
+|-----------:|---------------------:|
+| us west3-b | Salt Lake city, Utah |
+| us east4-b |    Ashburn, Virginia |
```
Comment (sonyafenge) [inline resource-management/docs/performance/perf-results.md:39]: for 5M official test, we have 5 simulator: us-west3-b,us-west3-b,us-central1-b,us-east1-c,us-central1-a

Comment (Sindica) [reply resource-management/docs/performance/perf-results.md:39]: us-west3-b twice?

Comment (Sindica) [reply resource-management/docs/performance/perf-results.md:39]: ./docs/setup-guide/setup-grs-test.md:export SERVER_ZONE=us-central1-a SIM_ZONE=us-central1-a,us-east1-b,us-west2-a,us-west4-a,us-west3-c CLIENT_ZONE=us-west3-b,us-east4-b

Can this be more specific?

**Review by yb01 — APPROVED** [review]
/lgtm

**Pull request merged**